{"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"abstract":[{"type":"text","text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call."}],"metadata":{"roleHeading":"Case","role":"symbol","modules":[{"name":"PasswordAndNfinityFramework"}],"title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO05cloudF0yACSSSg_AEtcACmF","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"cloudError","kind":"identifier"},{"kind":"text","text":"("},{"text":"errorCode","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?, ","kind":"text"},{"text":"errorMessage","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?)","kind":"text"}],"symbolKind":"case"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)"},"kind":"symbol","primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cloudError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"errorCode","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?, ","kind":"text"},{"kind":"externalParam","text":"errorMessage"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?)","kind":"text"}],"languages":["swift"],"platforms":["Mac Catalyst"]}],"kind":"declarations"},{"parameters":[{"content":[{"type":"paragraph","inlineContent":[{"text":"An optional error code as a string, that helps identify the type of error.","type":"text"}]}],"name":"errorCode"},{"content":[{"type":"paragraph","inlineContent":[{"text":"An optional human-readable message providing more details about the error.","type":"text"}]}],"name":"errorMessage"}],"kind":"parameters"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)"]}],"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"title":"PasswordAndNfinityFramework","kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","url":"\/documentation\/passwordandnfinityframework","type":"topic","role":"collection","abstract":[]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"NfinityCloudError","kind":"identifier"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","kind":"symbol","abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"title":"NfinityCloudError","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"NfinityCloudError"}]},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"type":"topic","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"cloudError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errorCode"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?, ","kind":"text"},{"kind":"externalParam","text":"errorMessage"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?)","kind":"text"}],"role":"symbol"}}}